packages/testcontainers/src/chains/reg_test_container/masternode.ts

```diff
@@ -119,7 +122,7 @@ export class MasterNodeRegTestContainer extends RegTestContainer {
       }
       await this.generate(1)
       return false
-    }, timeout, 1)
+    }, timeout, 100)
```

Comment:
Possible to be lower? So our test/ci can run faster.

Reply:
Not possible, because only 1 block can be mined per second. 100ms loop allowance allow it to mine forward if within consensus time limit.